doc/user/content/sql/create-sink.md

```diff
@@ -31,14 +31,15 @@ _item&lowbar;name_ | The name of the source or view you want to send to the sink
 
 ### Kafka connector
 
-{{< diagram "kafka-connector.svg" >}}
+{{< diagram "sink-kafka-connector.svg" >}}
 
 Field | Use
 ------|-----
 **KAFKA BROKER** _host_ | The Kafka broker's host name.
 **TOPIC** _topic&lowbar;prefix_ | The prefix used to generate the Kafka topic name to create and write to.
 **WITH OPTIONS (** _option&lowbar;_ **)** | Options affecting sink creation. For more details see [`WITH` options](#with-options).
 **CONFLUENT SCHEMA REGISTRY** _url_ | The URL of the Confluent schema registry to get schema information from.
+**KEYS (** _key&lowbar;column&lowbar;list_ **)** | An optional list of columns to use for the Kafka key.
```

Comment:
This is likely my own ignorance w/r/t Kafka, but would it be useful to describe what this does in Materialize in the details section of this page?

Reply:
It doesn't do anything in Materialize per se - only in Kafka.

In Kafka every record has a key and a value; before this change, the key would always be "null" in sinks that we produce. Now the key is composed of whatever values from the record that the user specifies in the `KEYS` clause (which we are renaming to `KEY`, btw - I will update this PR accordingly).

Do you know of any better way to express it?

Reply:
Ahh, that makes sense. I would assume folks actually running Kafka to understand that concept, so strike my comment.